source/core.py

```diff
@@ -946,6 +945,7 @@ def _doPostNvdaStartupAction():
 			)
 		except:
 			pass
+	_terminate(nvwave)

```

Comment:
Why was it moved? Could you update the initial description to give the reason if it is needed?